x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/installation_and_upgrade.md

```diff
@@ -815,6 +821,109 @@ And the Investigation Guide tab should NOT be displayed
 
 TODO: add scenarios https://github.com/elastic/kibana/issues/166215
 
+### Rule upgrade workflow: simple diff algorithm
+
+#### **Scenario: Rule field doesn't have an update and has no custom value**
+
+**Automation** 1 integration test
+
+```Gherkin
+Given at least 1 prebuilt rule is installed in Kibana
+And for this rule there is a new version available
+And this rule has an existing base version
```

Comment:
nit: Perhaps we can omit this line since it's implied from the line below and table that it exists